blog/002_packrat_layout.md

```diff
@@ -0,0 +1,129 @@
+# Packrat
+
+### Layout and Styles
+
+### The Goal
+ We want to setup a quick layout that lets us act like power users for our inventory management. To do this we're going to take a look at loading some custom fonts, using a prebuilt spiritual successor to bootstrap called tailwind css, and some clojure-land specific markup syntax.
+
+### Tailwind CSS
+[TailwindCSS](https://tailwindcss.com/)is a CSS framework based on utility classes. This means that instead of trying to handroll a lot of our own css classes and define styles for them, we choose from a whole lot of pre-written ones! This can be done by adding classes to your html which match the tailwind classes. For example
+
+### Lets start!
+If you're at all like me you get very frustrated and discouraged if something doesn't work right when you're trying to set it up. Therefore I like to start with something that works. So lets get a basic layout to start! We're going to follow their process for installation from the Tailwind [Install Docs](https://tailwindcss.com/docs/installation#installing-tailwind-as-a-post-css-plugin)
+```
+$ npm install tailwindcss autoprefixer postcss
+```
+and we can update our `public/index.html` to pull in a css file we will generate using our new deps!
+```
+// public/index.html
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Packrat</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+...
+```
+
+Then we should init a config file using this command
+```
+$ npx tailwindcss init
+```
+This creates a new config file for use with the tailwindcss cli tool which should look like this.
+```
+module.exports = {
+  purge: []
+  darkMode: false, // or 'media' or 'class'
+  theme: {
+    extend: {},
+  },
+  variants: {
+    extend: {},
+  },
+  plugins: [],
+}
+```
+
+Since we want to pre-process the css and shake out all the styles we don't use to minimize the amount of css we need to
```

Comment:
This can probably come after when we start talking about our production release setup instead of when we get it working